save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPluginConfig.kt

```diff
@@ -13,11 +18,13 @@ import kotlinx.serialization.Serializable
  *
  * @property execCmd a command that will be executed to mutate test file contents
  * @property destinationFileSuffix [execCmd] should append this suffix to the file name after mutating it.
+ * @property resourceNamePattern regex for the name of the test files
  */
 @Serializable
 data class FixPluginConfig(
     val execCmd: String,
     val destinationFileSuffix: String? = null,
+    val resourceNamePattern: Regex? = null
```

Comment:
wtf is that? 
We should have simple thing for resources: `destinationFileSuffix` and `testFileSuffix`